pkg/logutil/log.go

```diff
@@ -0,0 +1,118 @@
+package logutil
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/weaveworks/common/logging"
+	"github.com/weaveworks/common/server"
+)
+
+var (
+	// Logger is a shared go-kit logger.
+	// TODO: Change all components to take a non-global logger via their constructors.
+	// Prefer accepting a non-global logger as an argument.
+	Logger = log.NewNopLogger()
```

Comment:
Thanks for you effort! I wonder whether we should rather get rid off this logger instead as the todo suggest. What do you think?